lib/init.php

```diff
@@ -0,0 +1,4 @@
+<?php
+namespace Trendwerk\Smtp;
+
+$theme = new Smtp();
```

Comment:
- What is `$theme`?
- Should this plugin auto-initialize? This would probably break bc.